### PR TITLE
Update to 0.9.32

### PR DESCRIPTION
**Pull request by mike1729**
As above.

### CODE REVIEW

File: fork-off/Cargo.toml [hunk 1]
```diff
@@ -19,10 +19,10 @@ serde = "1"
 serde_json = "1"
 tokio = { version = "1.0", features = ["full"] }
 
-sp-core = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.31" }
-frame-support = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.31" }
-frame-system = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.31" }
-pallet-balances = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.31" }
+sp-core = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.32" }
```
Comment (timorl) [inline fork-off/Cargo.toml:22]: Forgot to bump the version of fork-off itself?

File: pallets/support/src/migration.rs [hunk 2]
```diff
@@ -3,6 +3,7 @@ use frame_support::{
     codec::{Decode, Encode},
     sp_io,
     sp_std::vec::Vec,
+    sp_std::vec::Vec,
```
Comment (timorl) [inline pallets/support/src/migration.rs:6]: Uh?

Comment (kostekIV) [reply pallets/support/src/migration.rs]: oh

File: fork-off/Cargo.toml (hunk 1 above)
Comment (kostekIV) [reply fork-off/Cargo.toml:22]: fixed

File: bin/node/Cargo.toml [hunk 3]
```diff
@@ -26,50 +26,50 @@ hex-literal = "0.3"
...
+sp-consensus-aura = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.32" }
+sp-consensus = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.32" }
+sc-consensus = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.32" }
+sc-client-api = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.32" }
+sp-runtime = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.32" }
+sp-timestamp = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.32" }
+pallet-staking = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.32" }
+try-runtime-cli = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.32", optional = true }
+sc-consensus-slots = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.32" }
 sp-arithmetic = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v0.9.31" }
```
Comment (h4nsu) [inline bin/node/Cargo.toml:55]: Why is this dependency left with `9.31`?

**Review by timorl — APPROVED** [review]

**Review by h4nsu — APPROVED** [review]

**Pull request merged**